src/deprecation.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2017 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef ZCASH_DEPRECATION_H
+#define ZCASH_DEPRECATION_H
+
+// Deprecation policy is 4th third-Tuesday after a release
+static const int APPROX_RELEASE_HEIGHT = 115000;
```

Comment:
Did you review the whole PR? I updated the release process to require that these values are updated, so it will always be the case that this value occurs after the release day.